src/PyCall.jl

```diff
@@ -30,6 +30,8 @@ end
 
 #########################################################################
 
+isfile(joinpath(dirname(@__FILE__), "..", "deps", "deps.jl")) ||
+    error("PyCall not properly installed. Please run Pkg.build(\"PyCall\")")
 include(joinpath("..", "deps", "deps.jl")) # generated by Pkg.build("PyCall")
```

Comment:
Should this line be using `dirname(@__FILE__)` too?  Is there some reason a relative path wouldn't work here?